src/shared/proto/primitive.proto

```diff
@@ -8,17 +8,20 @@ message Primitive
 {
     oneof primitive
     {
-        ChipPrimitive chip                    = 1;
-        KickPrimitive kick                    = 2;
-        MovePrimitive move                    = 3;
-        SpinningMovePrimitive spinning_move   = 4;
-        AutochipMovePrimitive autochip_move   = 5;
-        AutokickMovePrimitive autokick_move   = 6;
-        StopPrimitive stop                    = 7;
-        DirectControlPrimitive direct_control = 8;
+        EstopPrimitive estop                  = 1;
```

Comment:
lets add a quick test to this PR to make sure `EstopPrimitive` tag is first in this msg as we discussed before offline

I think its quick enough that it can be a part of this PR